tests/system_tests/backend_fixtures.py

```diff
@@ -171,7 +171,17 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
     def make_user(self, name: str | None = None, admin: bool = False) -> str:
         """Create a new user and return their username."""
-        name = name or f"user-{self.worker_id}-{random_string()}"
+        name = name or f"user{self.worker_id}{random_string()}"
```

Comment:
The dashes were important before to be able to separate out the worker ID from the random string.